miden/README.md

```diff
@@ -169,102 +174,65 @@ assert_eq!(vec![12586269025], outputs);
 ```
 Above, we used public inputs to initialize the stack rather than using `push` operations. This makes the program a bit simpler, and also allows us to run the program from arbitrary starting points without changing program hash.
 
-This program is rather efficient: the stack never gets more than 4 items deep. For some benchmarks of executing this program on the VM see [here](../README.md#Performance).
-
-## Crate features
-Miden VM can be compiled with the following features:
-
-* `std` - enabled by default and relies on the Rust standard library.
-* `concurrent` - implies `std` and also enables multi-threaded proof generation.
-* `no_std` does not rely on the Rust standard library and enables compilation to WebAssembly.
+## CLI interface
+If you want to execute, prove, and verify programs on Miden VM, but don't want to write Rust code, you can use Miden CLI. It also contains a number of useful tools to help analyze and debug programs.
 
-To compile with `no_std`, disable default features via `--no-default-features` flag.
+### Compiling Miden VM
+First, make sure you have Rust [installed](https://www.rust-lang.org/tools/install). The current version of Miden VM requires Rust version **1.62** or greater.
 
-### Concurrent proof generation
-When compiled with `concurrent` feature enabled, the VM will generate STARK proofs using multiple threads. The number of threads can be configured via `RAYON_NUM_THREADS` environment variable, and usually defaults to the number of logical cores on the machine. For benefits of concurrent proof generation check out these [benchmarks](../README.md#Performance).
-
-
-## Examples
-This crate contains several examples of Miden VM programs and demonstrates how these programs can be executed on Miden VM.
-
-## Running examples
-To run examples of executing programs and verifying proofs of their execution, do the following:
-
-First, compile an optimized version of the `miden` binary by running:
+Then, to compile Miden VM into a binary, run the following command:
 ```
 cargo build --release --features executable
 ```
-Or, if you want to compile the with multi-threaded support enabled, run:
+This will place `miden` executable in the `./target/release` directory.
+
+By default, the executable will be compiled in the single-threaded mode. If you would like to enable multi-threaded proof generation, you can compile Miden VM using the following command:
```

Comment:
It might be useful to include an example for WebAssembly as well

Reply:
We can't compile the executable to WASM as `executable` feature requires `std`. I agree that having a WASM example would be useful, but I think that's a more involved thing where we need to show how to do bindings to JS etc.

Reply:
Ah I see. I think I misunderstood based on what was written further below, but yeah, that makes sense. Thanks for clarifying